lib/publiccloud/provider.pm

```diff
@@ -623,11 +623,13 @@ c<{"key": "value"}> where c<value> must be a string.
 sub terraform_param_tags
 {
     my ($self) = @_;
+    my $openqa_var_server = get_var('OPENQA_URL', get_var('OPENQA_HOSTNAME'));
+    $openqa_var_server =~ s@^https?://|/$@@gm;
```

Comment:
keep in mind that `$openqa_var_server` can be just FQDN . I assume this will not be a problem here , right ?

Reply:
I hope not.
![image](https://github.com/os-autoinst/os-autoinst-distri-opensuse/assets/1254493/9b3dcf1f-7e72-40b0-b6ed-65dcd7bf6f5f)